docs/api-reference/next/image.md

```diff
@@ -162,6 +167,15 @@ When true, the image will be considered high priority and
 Should only be used when the image is visible above the fold. Defaults to
 `false`.
 
+### placeholder
+
+Setting `placeholder="blur"` enables an automatic placeholder effect for the image, 
+which is rendered on the page until the image is fully loaded.
+
+Blurry placeholder images are generated automatically for **statically imported** images.
```

Comment:
This isn't true though, the user still needs to add `placeholder=blur`. 

cc @Joonpark13 @spanicker

Reply:
Sorry, maybe that's confusingly worded. The first line says that you have to enable `placeholder=blur` the second line is just that the generation happens automatically for static images. you have to do your own generation for other images. Let me see if I can rephrase

Reply:
Ok I reworded that--hopefully it's more clear.